example-monorepos/with-nativewind/packages/app/features/home/screen.tsx

```diff
@@ -0,0 +1,69 @@
+import { View } from "react-native"
+import { A, H1, P, Text, TextLink } from 'app/design/typography'
+import { Row } from 'app/design/layout'
+
+import { MotiLink } from 'solito/moti'
+
+export function HomeScreen() {
+  return (
+    <View className="flex-1 justify-center items-center p-2">
+      <H1>Welcome to Solito.</H1>
+      <View className="max-w-xl">
+        <P className="text-center text-red text-purple">
```

Comment:
The `text-red` and `text-purple` did not work for my example. Using `text-indigo-500` did however, I did also bump my tailwind version to ^3.1.8

Reply:
`text-red` and `text-purple` are not valid Tailwind classNames. https://tailwindcss.com/docs/text-color